src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/group_convolution_backprop_data.cpp

```diff
@@ -180,7 +180,7 @@ INSTANTIATE_TEST_SUITE_P(smoke_GroupConvBackpropData_2d_AutoPadding_output_paddi
 /* ============= _3d GroupConvolution ============= */
 const std::vector<std::vector<ov::Shape>> input_shapes_3d = {{{1, 16, 5, 5, 5}},
                                                              {{1, 32, 5, 5, 5}}};
-const std::vector<std::vector<size_t >> kernels_3d = {{1, 1, 1}, {3, 3, 3}};
```

Comment:
As discussed, could you please return the parameters to the initial state.

Reply:
Applied.